### DISCUSSION TITLE
Problemas com site

### DISCUSSION THREAD

**Discussion by Bruno12N** [community/community#131846] [Programming Help]
### Select Topic Area

General

### Body

Bom dia, Gostaria de estar pedindo ajuda aos mais experientes! Estou com algumas dificuldades para fazer o deploy, ele realmente é demorado ou somente comigo que está demorando 3 a 4 dias para aparecer no google. Após esses dias ainda tenho que solicitar a indexação da pagina que demora mais 3 dias. Porém minha real duvida seria qual site é melhor para hospedagem tentei o GCP( Google Cloud Plataform) porém mesmo após de criar o bucket, tornar publico e esperar os 3 dias ele não aparecia com a URL do dominio, então fui para o netfly, que não sei se está me atendendo muito bem? Alguem tem alguma indicação?

**Comment by LiteBrite82** [comment]
Hi @Bruno12N 
I dont have a ton of experience with different hosting services. Heroku, netlify, GoDaddy. 
The more important thing to do to show up on google is having a good [robots.txt file](https://developers.google.com/search/docs/crawling-indexing/robots/intro#:~:text=A%20robots.txt%20file%20tells,web%20page%20out%20of%20Google.) and making sure you have a high [SEO score](https://developers.google.com/search/docs/fundamentals/seo-starter-guide).

**Reply by Bruno12N** [reply]
Hi @LiteBrite82
Thank you! Your tips are very valuable. But I need more help, my website was not approved by Google Ad Sense, Could you give your opnion? https://cdicalc.com.br/ 
(I tried to practice my English lol)

**Reply by LiteBrite82** [reply]
Im not familiar with ad sense. I ran a lighthouse report on your page; your scores are pretty good:
<img width="1667" alt="Screenshot 2024-07-12 at 9 47 13 AM" src="https://github.com/user-attachments/assets/4acdcfa4-6ff9-4de6-9922-b64ccbc0f526">

**Reply by LiteBrite82** [reply]
[How to use lighthouse](https://www.youtube.com/watch?v=GBTq5AtgpaQ)

**Reply by Bruno12N** [reply]
How do you know so much about programming? Could you give me some study tips? I'm at the Faculty of Systems Analysis and Development, but not taking classes at the moment

**Reply by Bruno12N** [reply]
Thanks for lighthouse report!!

**Reply by LiteBrite82** [reply]
I did a full-stack web dev bootcamp during the covid pandemic. Having a live instructor was invaluable for me and my learning style. Google search, stack overflow, and YouTube are my go to tools. for me, coding is 90% research 10% coding. 

My best advice is to make friends; I still hang out with folks I went to the coding camp with - we all have helped each other get jobs over the years. 

As far as studying goes... tech changes so fast; by the time you have something memorized - it becomes obsolete. Study the general concepts; dig into the documentation of the tech you are working with at the moment. 

After school/classes its more about certificates; you learn a ton pursuing certificates. 

Figure out which field you have a passion for and let your curiosity guide you. backend, front end, AI / ML, design, etc...

I'm happy to help!! Good luck!!! Reach out to the community any time!
